src/modules/formBuilder/components/FormBuilderHeader.tsx

```diff
@@ -50,15 +52,23 @@ const FormBuilderHeader: React.FC<FormEditorHeaderProps> = ({
       }),
     });
 
+  const isFirstDraft = formDefinition.version === '0';
```

Comment:
Curious to hear your thoughts on deriving this from the form version -- maybe it's a bit brittle. We could also add something to the schema coming from the backend if that feels more certain, but I wasn't sure it was worth the extra API complexity.

Reply:
Yeah this seems fine